src/django/api/services/moderation_events_service.py

```
@@ -0,0 +1,57 @@
+
+    @staticmethod
+    def fetch_moderation_event_by_uuid(uuid):
+        try:
+            return ModerationEvent.objects.get(uuid=uuid)
+        except ModerationEvent.DoesNotExist:
+            raise NotFound(
+                create_error_detail(
+                    field="moderation_id",
+                    detail="Moderation event not found."
+                )
+            )
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)